recipes/jupyterlab-lsp/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "jupyter-lsp" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3abe20ec48a3f6b132653abe597af2b6cd0005539ab0d74e0ba203927124d333
+
+build:
+  noarch: python
+  number: 0
+
+outputs:
+  - name: {{ name }}
+    script: build_base.bat  # [win]
+    script: build_base.sh  # [not win]
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - notebook >=4.3.1
+        - python
+        - setuptools
+    test:  # Seems ignored by conda-build
+      requires:
+        - jupyterlab =2
+        - nodejs >=8,<13
```

Comment:
Proposed:
```
- nodejs >=10
```
`10` is the current LTS, and it also seems to build fine with `13` on the various platforms.